packages/nimble-components/src/rich-text-editor/tests/rich-text-editor-labels.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { html } from '@microsoft/fast-element';
+import { richTextEditorTag, type RichTextEditor } from '..';
+import { type Fixture, fixture } from '../../utilities/tests/fixture';
+import { themeProviderTag, type ThemeProvider } from '../../theme-provider';
+import {
+    LabelProviderRichTextEditor,
+    labelProviderRichTextEditorTag
+} from '../../label-provider/rich-text-editor';
+import { RichTextEditorPageObject } from '../testing/rich-text-editor.pageobject';
+import { ToolbarButton } from '../testing/types';
+
+async function setup(): Promise<Fixture<ThemeProvider>> {
+    return fixture<ThemeProvider>(
+        html`
+      <${themeProviderTag}>
+          <${labelProviderRichTextEditorTag}></${labelProviderRichTextEditorTag}>
+          <${richTextEditorTag}></${richTextEditorTag}>
+      <${themeProviderTag}>`
+    );
+}
+
+describe('Rich Text Editor with LabelProviderRichTextEditor', () => {
+    let element: RichTextEditor;
+    let labelProvider: LabelProviderRichTextEditor;
+    let connect: () => Promise<void>;
+    let disconnect: () => Promise<void>;
+    let pageObject: RichTextEditorPageObject;
+
+    beforeEach(async () => {
+        let themeProvider: ThemeProvider;
+        ({ element: themeProvider, connect, disconnect } = await setup());
+        element = themeProvider.querySelector(richTextEditorTag)!;
+        labelProvider = themeProvider.querySelector(
+            labelProviderRichTextEditorTag
+        )!;
+        pageObject = new RichTextEditorPageObject(element);
+    });
+
+    afterEach(async () => {
+        await disconnect();
+    });
+
+    it('uses correct labels for bold button', async () => {
+        await connect();
+        labelProvider.toggleBold = 'Bold';
```

Comment:
nit: To represent more that this string is customized app specific label string, can we have `Customized Bold Label`

Consider applying for all instances.

Reply:
Updated